tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -299,6 +299,157 @@ def try_get_features():
+        ],
+        entities=[Entity("customer_id2", ValueType.INT64)],
+        max_age=Duration(seconds=3600)
+    )
+
+    client.set_project(PROJECT_NAME)
+    client.apply(customer_fs)
+
+    customer_fs = client.get_feature_set(name="customer2")
+    client.ingest(customer_fs, list_type_dataframe, timeout=600)
+    time.sleep(15)
+
+    online_request_entity = [{"customer_id2": 0},{"customer_id2": 1}]
+    online_request_features = [
+        "customer2_rating",
+        "customer2_cost",
+        "customer2_past_transactions_int",
+        "customer2_past_transactions_double",
+        "customer2_past_transactions_float",
+        "customer2_past_transactions_string",
+        "customer2_past_transactions_bool"
+    ]
+
+    online_features_actual = client.get_online_features(entity_rows=online_request_entity, feature_refs=online_request_features)
+    online_features_expected = {
+        "customer_id2": [0,1],
+        "customer2_rating": [0,1],
+        "customer2_cost": [0.5,1.5],
+        "customer2_past_transactions_int": [[0,2],[1,3]],
+        "customer2_past_transactions_double": [[0.5,2.0],[1.5,3.0]],
+        "customer2_past_transactions_float": [[0.5,2.0],[1.5,3.0]],
+        "customer2_past_transactions_string": [['first_0','second_0'],['first_1','second_1']],
+        "customer2_past_transactions_bool": [[True,False],[True,False]]
+    }
+
+    # Case 2: List entity check
+    district_fs = FeatureSet(
+        name="district",
+        features=[
+            Feature(name="district_rating", dtype=ValueType.INT64),
+            Feature(name="district_cost", dtype=ValueType.FLOAT),
+            Feature(name="district_past_transactions_int", dtype=ValueType.INT64_LIST),
+            Feature(name="district_past_transactions_double", dtype=ValueType.DOUBLE_LIST),
+            Feature(name="district_past_transactions_float", dtype=ValueType.FLOAT_LIST),
+            Feature(name="district_past_transactions_string", dtype=ValueType.STRING_LIST),
+            Feature(name="district_past_transactions_bool", dtype=ValueType.BOOL_LIST)
+        ],
+        entities=[Entity("district_ids", dtype=ValueType.INT64_LIST)],
+        max_age=Duration(seconds=3600)
+    )
+
+    client.set_project(PROJECT_NAME)
+    client.apply(district_fs)
+
+    district_fs = client.get_feature_set(name="district")
+    client.ingest(district_fs, entity_list_type_dataframe, timeout=600)
+    time.sleep(15)
+
+    online_request_entity2 = [{"district_ids": [np.int64(1),np.int64(2),np.int64(3)]}]
+    online_request_features2 = [
+        "district_rating",
+        "district_cost",
+        "district_past_transactions_int",
+        "district_past_transactions_double",
+        "district_past_transactions_float",
+        "district_past_transactions_string",
+        "district_past_transactions_bool"
+    ]
+
+    online_features_actual2 = client.get_online_features(entity_rows=online_request_entity2, feature_refs=online_request_features2)
```

Comment:
Unfortunately, I think you need to poll `get_online_features` for feature values as they might not be immediately available. See [here](https://github.com/feast-dev/feast/blob/b461f4dea517de5e5ad7c9114e069081d9437810/tests/e2e/redis/basic-ingest-redis-serving.py#L261) for how to do it.